lib/api/core/validation/types/type.js.template

```diff
@@ -15,6 +15,7 @@ class MyNewType extends BaseType {
    * @param {TypeOptions} typeOptions
    * @param {*} fieldValue
    * @param {string[]} errorMessages
+   * @return {boolean}
```

Comment:
Maybe you can also add the `require` for `PreconditionError` because this will be used in any new type defined